### PR TITLE
CSV attachment viewer (#438)

### PR DESCRIPTION
**Pull request by VikomiC**
- added striped view for table.

### CODE REVIEW

**Comment by just-boris** [comment]
Build failed because of #527 
If you are making visual improvements, I'd like to see them in action. Please add csv attachment into our test data here: https://github.com/allure-framework/allure-core/blob/master/allure-model/src/test/resources/allure-results/c32503a8-2981-43da-afa5-e0d82dacfefd-testsuite.xml#L102
Then it will be included into report after build.

**Comment by VikomiC** [comment]
image

Now I'm working on adding this csv to allure-model.

**Comment by eroshenkoam** [comment]
retest this please

**Comment by arpitgold** [comment]
Is this also works for *.xlsx file?

**Comment by just-boris** [comment]
@arpitgold no, just `.csv` for now.

**Comment by just-boris** [comment]
retest this please

**Comment by just-boris** [comment]
sorry for bothering you, guys, but we are trying to show report example after build on jenkins, as it was in teamcity

**Comment by just-boris** [comment]
Brilliant. Now we can see the demo report in build artifacts. To do it, go to "Details" on build status block, then click on "Build Artifacts" -> "allure-report" -> "index.html"

File: allure-report-face/src/main/webapp/templates/testcase/attachment.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
             <pre><code>{{attachText}}</code></pre>
         </div>
         <div ng-switch-when="csv">
-            <table class="test-list table table-hover table-condensed">
+            <table class="test-list table table-hover table-striped table-condensed">
```
Comment (just-boris) [inline allure-report-face/src/main/webapp/templates/testcase/attachment.html]: Could you please remove the `table-hover` class? It causes hover effect on table rows, but we are using it only for clickable items.

**Comment by VikomiC** [comment]
How can we make retest for it ?
I added csv files for testing in the allure-model, as was requested. It is located in the place with all attachments.

File: allure-report-face/src/main/webapp/templates/testcase/attachment.html (hunk 1 above)
Comment (VikomiC) [inline allure-report-face/src/main/webapp/templates/testcase/attachment.html]: Ok, I'll do it.

**Comment by just-boris** [comment]
As you can see, build will be run for each commit.

**Comment by VikomiC** [comment]
Ok, so, it is run again :) Should be perfect now ;)

**Pull request merged**